index.html

```diff
@@ -5,19 +5,20 @@
     <meta http-equiv="X-UA-Compatible" content="IE=edge" />
     <meta name="viewport" content="width=device-width, initial-scale=1.0" />
     <title>Mad Libs Word Game - Chingu Voyage 43 - Group 9</title>
+    <link rel="stylesheet" href="index_mobile.css" />
   </head>
-  <body>
-    <header>
+  <body class="flex">
+    <header class="flex__top">
```

Comment:
In this line we specify that the `<header>` element has a class of `flex__top`

Reply:
In the css file we do not have a `flex__top class so this part gets ignored...

Reply:
or does it? Let's look at our css once again...
```
.flex,
.flex > *,
.flex__middle--inner,
.flex__bottom {
  display: flex;
  flex-direction: column;
  align-items: center;
  text-align: center
}
```

Reply:
More specifically look at `.flex > *,`

Reply:
`.flex` is to target class of `flex` `>` means the child and what comes next is the targeted `<element>` and in this case we have specified `*` meaning all.